README.md

```diff
@@ -138,6 +138,38 @@ Tools we do not currently use but are interested in evaluating:
 * [Reek](https://github.com/troessner/reek) and [Flog](https://github.com/seattlerb/flog) if they're not already included by CodeClimate or other gems (Marri's brain thinks they might be, the way flay is in duplication, but isn't citing its sources)
 * [Reek's brother and sister gems](https://github.com/troessner/reek#brothers-and-sisters)
 
+### Backstop - UI validation
+
+We use [BackstopJS](https://github.com/garris/BackstopJS) to run regression tests on the UI + CSS.
+
+Before each test, we automatically run `script/before_backstop.rb` to set up a consistent environment.
+To perform a test:
+
+```bash
+# you may wish to recreate + reseed your database for consistent images:
+# bin-docker/rails db:drop db:setup
+
+# set the layout you want to validate
+export LAYOUT=starrylight
+
+# switch to the branch you want to use as a reference
+git switch main
+# create reference images and save them to backstop/
+bin-docker/backstop reference $LAYOUT
```

Comment:
i GUESS you can do it with env variables xD

Reply:
I just wanted to demonstrate all the locations that you have to change if you change the layout XD you can do it indivudally if you want